ddtrace/contrib/pyramid/patch.py

```diff
@@ -37,3 +39,20 @@ def traced_init(wrapped, instance, args, kwargs):
 
     wrapped(*args, **kwargs)
     trace_pyramid(instance)
+
+def insert_tween_if_needed(settings):
+    if 'pyramid.tweens' not in settings:
+        return
+    tweens_list = settings['pyramid.tweens']
+    # If the list is empty, pyramid does not consider the tweens have been
+    # set explicitely so we will insert the tween with add_tween.
+    if not tweens_list.strip():
```

Comment:
is it a list or a string? a list doesn't have `strip()`, so I'm guessing it is not.

Reply:
It is a string representing a list indeed, the naming is misleading. Will change that.